.github/workflows/weekly-resources-clean.yml

```diff
@@ -33,5 +33,10 @@ jobs:
 
       - name: Clean old ec2
         run: go run tools/workflow/clean_ec2.go
```

Comment:
Does this line need to be deleted?

Reply:
Forgot to push it haha. Thanks for reminding me.